monitoring-datastores/postgresql/main.tf

```diff
@@ -0,0 +1,57 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+provider "aws" {
+  region = var.aws_region
+}
+
+module "monitor_postgresql_datastore_lambda_function" {
+  source = "terraform-aws-modules/lambda/aws"
+  
+  function_name = "monitor-postgresql-datastore-lambda"
+  description   = "Lambda function that monitors postgresql datastore"
+  handler       = "monitor-postgresql-datastore.main"
+  runtime       = "python3.10"
+
+  publish                  = true
+  architectures = ["arm64"]
```

Comment:
do we have to specify arch?

Reply:
We build the code on docker then push it for lambda.
In the docker build process we downloads a pip lib called psycopg2.
For some reason it downloads it for arm64 so I must run the lambda on it as well.

Maybe for customer the docker build will be different. if so he can just delete that line.